include/alp/reference/blas1.hpp

```diff
@@ -2847,28 +2847,85 @@ namespace alp {
 	}
 
 	/**
-	 * Sort vectors, function available to user, e.g. to sort eigenvectors
+	 * Compute the permutation vector needed to sort the input vector according to
+	 * the provided \a cmp function.
```

Comment:
It seems that this function uses a hard-coded comparison function - it is not provided by the caller.

Reply:
This will be addressed soon by allowing a single variant of the function which always expects a poset/total order.